### PR TITLE
User login unit test

### PR DESCRIPTION
**Pull request by yang445**

### CODE REVIEW

File: test_user.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import User
```
Comment (teusday) [inline test_user.py]: If you make it import User from User, then you won't need to use User.User later

File: test_user.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        self.testUser = User.User("TestUser", "password123")
+
+    def test_correct_credentials(self):
+        self.assertTrue(self.testUser.login("password123"), "Did not accept the correct password")
+
+    def test_incorrect_credentials(self):
+        self.assertFalse(self.testUser.login("incorrect"), "Accepted the incorrect password")
+
+    def test_uppercase_credentials(self):
+        self.assertFalse(self.testUser.login("Password123"), "Accepted a password that incorrect capitalization")
```
Comment (teusday) [inline test_user.py]: Test for trailing spaces?

File: User.py [hunk 3]
```diff
@@ -1,32 +1,31 @@
...
-    	self.name = username
-    	self.pwd = pwd
-    	self.wishList = list()  # list of items
-    	self.accessList = list() # list of users with access
+    def __init__(self, username, pwd):  # create
+        self.name = username
+        self.pwd = pwd
+        self.wishList = list()  # list of items
+        self.accessList = list()  # list of users with access
         pass
```
Comment (Lightning0) [inline User.py]: Pass not necessary.

File: test_user.py (hunk 2 above)
Comment (leithouse) [reply test_user.py]: Not in the specs, spaces could be in the password

**Pull request merged**